test-psql2mysql.py

```diff
@@ -89,3 +89,21 @@ def test_scanTablefor4ByteUtf8Char(self, mock_utf8mb_rows,
         self.assertEqual(len(result), 1)
         self.assertEqual("name", wrong_item["column"])
         self.assertEqual("id=1", wrong_item["primary"][0])
+
+    @mock.patch('psql2mysql.DbWrapper.getTextColumns')
+    @mock.patch('psql2mysql.DbWrapper._query_long_text_rows')
+    def test_scanTableForLongTexts(self, mock_query_long_text_rows,
+                                   mock_getTextColumns):
+        table = FakeTable("test", [], [FakePrimary("id")])
+        mock_getTextColumns.return_value = ["text"]
+        mock_query_long_text_rows.return_value = [
+            {"text": u"a"*p2m.MAX_TEXT_LEN, "id": 1},
+            {"text": "name with Ä", "id": 2},
+            {"text": u"a"*(p2m.MAX_TEXT_LEN+1), "id": 3},
+            {"text": None, "id": 4}
+        ]
+        result = self.db_wrapper.scanTableForLongTexts(table)
+        wrong_item = result[0]
+        self.assertEqual(len(result), 1)
```

Comment:
Could you maybe switch those 2 lines (106 and 107)? I think it makes more sense to first make sure that the size is 1 and than access the first item.
I know it works for this case, but in case there were no error, test would fail with `list index out of range` which is a bit confusing at first